src/screens/VocabScreen.js

```diff
@@ -0,0 +1,112 @@
+/**
+  Map screen.
```

Comment:
Add `@flow` to get this file type-checked by Flow. Make sure to run Flow afterward and fix any errors.